infra/base/common/crs/kie.yaml

```
@@ -0,0 +1,26 @@
+apiVersion: app.kiegroup.org/v2
+kind: KieApp
+metadata:
+  name: rhpam-mercury
+spec:
+  auth:
+    sso:
+      adminPassword: RedHat1!
+      adminUser: admin
+      realm: mercury
+      disableSSLCertValidation: true
+      url: https://keycloak-theom.apps.mw-ocp4.cloud.lab.eng.bos.redhat.com/auth
```

This url depends on the namespace/lab so it won't work anywhere else.
For now change it to 
```suggestion
      url: https://keycloak-mercury-common.apps.mw-ocp4.cloud.lab.eng.bos.redhat.com/auth
```